geometry_msgs/Pose hole_pose_estimate
---
geometry_msgs/Pose hole_pose
